### DISCUSSION TITLE
The page never rebuilts with unstable_revalidate.

### DISCUSSION THREAD

**Discussion by yassinebridi** [vercel/next.js#13869] [Help]
I'm  trying to revalidate the page each new comments comes in. Comments are hosted on firebase, and i'm fetching them like this:
```js
const allComments = await firebase
  .collection('posts')
  .doc(params.slug as string)
  .collection('comments')
  .orderBy("date", "desc")
  .get()

const docs = await allComments.docs
const comments = await JSON.parse(JSON.stringify(docs))
```

and this is the getStaticProps return:
```js
return {
  props: {
    comments: comments,
    updated: updated
  },
  unstable_revalidate: 5,
};
```
i added a timestamp `updated` to get the time of the generated page
```js
const updated = new Date().toLocaleString();
```

still, the page never generate.

the `getStaticPaths` function is defined like this:
```js
export const getStaticPaths: GetStaticPaths = async () => {
  const posts = getAllPosts(["slug", "mainTag"]);
  return {
    paths: posts.map((posts) => {
      return {
        params: {
          slug: posts.slug,
          type: posts.mainTag,
        },
      };
    }),
    fallback: false,
  };
};
```

**Comment by VivekNeel** [comment]
Keep refreshing, it gets re generated. Am using the same, it works but my time is set to 1 sec

**Reply by yassinebridi** [reply]
it has been one hours now, still no new comments, no timestamp changes.

**Comment by givi-phirtskhalava** [comment]
I just deployed on Vercel and it works. Make sure you're using `next start`.

**Reply by yassinebridi** [reply]
it worked locally with: next build && next start.
but not in vercel. This is my package.json's scripts:
```json
"scripts": {
  "dev": "NODE_OPTIONS='--inspect' next dev",
  "build": "next build",
  "start": "next start",
  "clean": "rm -r .next && clear || clear"
},
```
